Rakefile

```diff
@@ -93,10 +93,15 @@ namespace :test do
   end
 end
 
+require 'rspec/core/rake_task'
+RSpec::Core::RakeTask.new(:spec) do |t|
+  t.pattern = Dir.glob('spec/**/*_spec.rb')
+end
```

Comment:
I'll leave that up to you (rails-api members) to decide. For me adding mrspec it's just another layer on top of what already works.